src/components/ens-registration/ConfirmContent/EditContent.tsx

```diff
@@ -0,0 +1,61 @@
+import lang from 'i18n-js';
+import React, { useCallback } from 'react';
+import { Switch } from 'react-native-gesture-handler';
+import ButtonPressAnimation from '../../animations/ButtonPressAnimation';
+import { Divider, Inline, Inset, Stack, Text } from '@rainbow-me/design-system';
+import { useNavigation } from '@rainbow-me/navigation';
+import Routes from '@rainbow-me/routes';
+import { colors } from '@rainbow-me/styles';
+
+const EditContent = ({
+  accentColor,
+  sendReverseRecord,
+  setSendReverseRecord,
+  showReverseRecordSwitch,
+}: {
+  accentColor: any;
+  sendReverseRecord: boolean;
+  setSendReverseRecord: React.Dispatch<React.SetStateAction<boolean>> | null;
+  showReverseRecordSwitch?: boolean;
+}) => {
+  const { navigate } = useNavigation();
+  const openPrimaryENSNameHelper = useCallback(() => {
+    navigate(Routes.EXPLAIN_SHEET, { type: 'ens_primary_name' });
+  }, [navigate]);
+
+  if (!showReverseRecordSwitch) return null;
```

Comment:
i think you could also do `return showReverseRecordSwitch && (<Inset>...</Inset>)

Reply:
i guess so, thought this might have been more readable